lib/kamal/configuration/docs/accessory.yml

```diff
@@ -98,3 +98,92 @@ accessories:
     # Defaults to kamal:
     network: custom
 
+    # Proxy
+    #
+    proxy:
+      # Hosts
```

Comment:
You should be able to replace all of this with:

```
proxy:
  ...
```

The validation and doc generation will use the config in `proxy.yml` in that case.